CMakeLists.txt

```diff
@@ -692,6 +693,10 @@ if (NOT MIOPEN_REQ_LIBS_ONLY)
     add_subdirectory(profiler)
 endif()
 
+if (CK_EXPERIMENTAL_BUILDER)
+  add_subdirectory(experimental/builder/test)
```

Comment:
I'd make a separate `experimental/builder/CMakeLists.txt` and add the test from there, just to make to avoid tainting the top-level CMakeLists.txt with any experimental stuff. We can also use that to export CMake targets etc eventually.

Reply:
Thanks, you are right. Made the fix.